bin/_tag.sh

```diff
@@ -28,7 +32,11 @@ head_root_tag() {
 
 clean_head_root_tag() {
     if clean_head ; then
-        echo "git-$(git_sha HEAD)"
+        if [ "$(named_tag)" != "undefined" ]; then
+            echo "$(named_tag)"
+        else
+            echo "git-$(git_sha HEAD)"
```

Comment:
realizing we only ever call `git_sha` as `git_sha HEAD`. what do you think about renaming it to `git_sha_head`, and hard-coding `HEAD` rather than accepting a param?